MekHQ/src/mekhq/campaign/mission/AtBDynamicScenarioFactory.java

```diff
@@ -557,100 +547,104 @@ public static ScenarioObjective translateTemplateObjective(AtBDynamicScenario sc
             actualObjective.addForce(forceName);
         }
-        
+
         for(String unitID : objectiveUnitIDs) {
             actualObjective.addUnit(unitID);
         }
-        
+
         // if the objective specifies that it's to reach or prevent reaching a map edge
         // and has been set to "force destination edge", set that here
-        if(actualObjective.getDestinationEdge() == OffBoardDirection.NONE && 
+        if(actualObjective.getDestinationEdge() == OffBoardDirection.NONE &&
                 calculatedDestinationZone != OffBoardDirection.NONE &&
                 (actualObjective.getObjectiveCriterion() == ObjectiveCriterion.ReachMapEdge ||
                 actualObjective.getObjectiveCriterion() == ObjectiveCriterion.PreventReachMapEdge)) {
             actualObjective.setDestinationEdge(calculatedDestinationZone);
         }
-        
+
         return actualObjective;
     }
-    
+
     /**
      * Scale the scenario's objective time limits, if called for, by the number of units
      * that have associated force templates that "contribute to the unit count".
      */
     private static void scaleObjectiveTimeLimits(AtBDynamicScenario scenario, Campaign campaign) {
         int primaryUnitCount = 0;
-        
+
         for(int forceID : scenario.getPlayerForceTemplates().keySet()) {
             if(scenario.getPlayerForceTemplates().get(forceID).getContributesToUnitCount()) {
                 primaryUnitCount += campaign.getForce(forceID).getAllUnits().size();
             }
         }
-        
+
         for(BotForce botForce : scenario.getBotForceTemplates().keySet()) {
             if(scenario.getBotForceTemplates().get(botForce).getContributesToUnitCount()) {
                 primaryUnitCount += botForce.getEntityList().size();
             }
         }
-        
+
         for(ScenarioObjective objective : scenario.getScenarioObjectives()) {
             if(objective.getTimeLimitType() == TimeLimitType.ScaledToPrimaryUnitCount) {
                 objective.setTimeLimit(primaryUnitCount * objective.getTimeLimitScaleFactor());
             }
         }
     }
-    
+
     /**
      * Handles random determination of light conditions for the given scenario, as per AtB rules
      * @param scenario The scenario for which to set lighting conditions.
      */
     private static void setLightConditions(AtBDynamicScenario scenario) {
-        int light = PlanetaryConditions.L_DAY;
-
         int roll = Compute.randomInt(10) + 1;
-        if (roll < 6) light = PlanetaryConditions.L_DAY;
-        else if (roll < 8) light = PlanetaryConditions.L_DUSK;
-        else if (roll == 8) light = PlanetaryConditions.L_FULL_MOON;
-        else if (roll == 9) light = PlanetaryConditions.L_MOONLESS;
-        else light = PlanetaryConditions.L_PITCH_BLACK;
-        
+        int light = PlanetaryConditions.L_DAY; //this handles all rolls less than 6
+
+        if ((6 <= roll) && (roll < 8)){
```

Comment:
Inconsistent spacing before the open brace in this block.

Reply:
The logic is also faulty, so I've reverted this change and fixed the formatting.